lua/nvim-treesitter/ts_utils.lua

```diff
@@ -208,7 +208,8 @@ function M.update_selection(buf, node, selection_mode)
 
   -- Convert exclusive end position to inclusive
   if end_col == 1 then
-    vim.fn.setpos(".", { buf, end_row - 1, -1, 0 })
+    local previous_row = vim.fn.col { end_row - 1, "$" } - 1
```

Comment:
Cna you change the variable name to make clear that previous_row is not a row but a column?

Reply:
Why wasn't the -1 working? Doesn't it have the same meaning for vim of taking the last character of that line?

Reply:
oops, yes, this is a column. Changed!

> Why wasn't the -1 working?

Any value lower than 1 is interpreted as 1, the docs from `:h setpos()` confirm this